### DISCUSSION TITLE
Social Images queue name

### DISCUSSION THREAD

**Discussion by SanderGo** [studio1902/statamic-peak-seo#21] [Ideas]
I would love to PR an update that allows you to (optionally) specify the queue name for the Social Images generation job. 

Do you want a discussion topic opened for minor updates like this as well? The PR placeholder messages says so, but might just be extra work for you sometimes? Either way is fine with me :)

**Comment by robdekort** [comment]
I prefer the discussions and appreciate you doing so. Don't want someone to do work when I know beforehand I won't want to merge something. You're one of the only few actually reading the message and doing so btw. Thanks!

I'm fine with that PR, sounds like a good move!

**Reply by SanderGo** [reply]
Cool, I understand! Not a problem at all :)
